src/medals/utils.cljs

```diff
@@ -0,0 +1,18 @@
+(ns medals.utils
+  (:require [reagent.core :as r]
+            [medals.constants :refer [COUNTRIESMEDALCOLORS]]))
```

Comment:
`COUNTRIESMEDALCOLORS` is badly named 😛 

- should be lowercase
- please use `-` as a separator
- `medals` is part of `medals.constants` so please not use `medals` as part of variable name